package.json

```diff
@@ -37,6 +37,7 @@
     "lint:fix": "eslint . --ext js,json --fix",
     "lint:changed": "{ git ls-files --others --exclude-standard ; git diff-index --name-only --diff-filter=d HEAD ; } | grep --regexp='[.]js$' --regexp='[.]json$' | tr '\\n' '\\0' | xargs -0 eslint",
     "lint:changed:fix": "{ git ls-files --others --exclude-standard ; git diff-index --name-only --diff-filter=d HEAD ; } | grep --regexp='[.]js$' --regexp='[.]json$' | tr '\\n' '\\0' | xargs -0 eslint --fix",
+    "lint:shellcheck": "shellcheck --version && find . -type f -name '*.sh' ! -path './node_modules/*' -print0 | xargs -0 shellcheck",
```

Comment:
I didn't notice this `shellcheck --version` at the beginning here earlier - this must be why it was included in `yarn lint` originally? Because it'd just skip this command if it wasn't installed. Ah well.

For CI, it'd probably be better for it to fail hard if `shellcheck` isn't found. Otherwise a failure in the install step might escape our notice.

Reply:
I actually added it as I thought it might handy for debugging. If it makes sense we could add a check for shellcheck in here as well? I was also thinking it might be useful to run shellcheck over each command in the `package.json` using `jq`... 

something like `jq -r .scripts < package.json | xargs shellcheck`

Reply:
Ah right, that makes sense. I looked at the `&&` and assumed it was meant to short-circuit.
Right, `shellcheck --version` will still return a non-zero exit code if it's not found, so it'll still register as a CI failure.